sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -45,7 +45,7 @@
     <dependency>
```

Comment:
The azure-communication-common dependency should be updated to 1.0.0-beta.2 as well

Reply:
@DominikMe This PR seems correct according to the release triggered [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=546748&view=logs&s=0208d850-5e0c-50f4-79f1-398f0bd9bc4b&j=11583220-a959-54f5-b85e-e18d6f78ad68).
Did we do a separate release for `azure-communication-common` ?

Reply:
@samvaity I don't remember if the release was done separately by Wes or Mitch. But we have released `azure-communication-common` with version `1.0.0-beta.1` 

https://search.maven.org/search?q=a:azure-communication*

Reply:
Updating here - https://github.com/Azure/azure-sdk-for-java/pull/15866